src/services/clientSideSortUtil.ts

```diff
@@ -0,0 +1,19 @@
+import { orderBy } from 'lodash';
+
+const ACCESS_ORDINALS = { DENY: 1, WRITE: 0, READ: -1 };
+
+const compareAccessLevels = (a, b) => {
+  return ACCESS_ORDINALS[a.accessLevel] - ACCESS_ORDINALS[b.accessLevel];
+};
+
+export const clientSideSort = (data, sortField, order, sortBy) => {
+  if (sortField === 'accessLevel') {
+    let sorted = data.sort(compareAccessLevels);
```

Comment:
I would avoid `Array.sort`, if possible since it does an in-place sort... if absolutely necessary, I'd do `[...data].sort` to avoid mutating original `data`